RecoLocalMuon/RPCRecHit/test/testRPCPointProducer.py

```diff
@@ -1,49 +1,54 @@
 import FWCore.ParameterSet.Config as cms
+process = cms.Process("OwnParticles")
 
-##process = cms.Process("RPCPointProducer")
-process = cms.Process("OWNPARTICLES")
-
-process.load("Geometry.MuonCommonData.muonIdealGeometryXML_cfi")
-process.load("Geometry.RPCGeometry.rpcGeometry_cfi")
-process.load("Geometry.CSCGeometry.cscGeometry_cfi")
-process.load("Geometry.DTGeometry.dtGeometry_cfi")
+process.load('Configuration.Geometry.GeometryExtended2018Reco_cff')
+process.load('Configuration.Geometry.GeometryExtended2018_cff')
 process.load("Geometry.MuonNumbering.muonNumberingInitialization_cfi")
-
+process.load("DQMServices.Components.MEtoEDMConverter_cfi")
+process.load("DQMServices.Core.DQM_cfg")
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
-process.GlobalTag.globaltag = "START53_V27::All"
-
-process.load("Configuration.StandardSequences.Services_cff")
-process.load("Configuration.StandardSequences.GeometryDB_cff")
-process.load("Configuration.StandardSequences.Reconstruction_cff")
 process.load("Configuration.StandardSequences.MagneticField_AutoFromDBCurrent_cff")
+process.GlobalTag.globaltag = "124X_dataRun3_Express_v5"
 
-process.load("FWCore.MessageService.MessageLogger_cfi")
 
-process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(10) )
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(2000) )
 
 process.source = cms.Source("PoolSource",
-    fileNames = cms.untracked.vstring(
-        #	'/store/data/Commissioning10/Cosmics/RECO/v3/000/127/155/005F9301-2E16-DF11-B60B-0030487CD6B4.root'
-        '/store/data/Run2012D/RPCMonitor/RAW/v1/000/207/900/1AFE2D3D-A836-E211-B86A-001D09F23D1D.root'
-        )                           
-)
+   fileNames = cms.untracked.vstring(
 
+# local copy for test only:
+'file:/eos/cms/store/group/dpg_rpc/comm_rpc/Sandbox/mileva/testRPCMon2022/1cb9263a-550d-4e86-92f3-f01574867137.root'
+)
+)
 
 process.load("RecoLocalMuon.RPCRecHit.rpcPointProducer_cff")
-process.rpcPointProducer.tracks = cms.InputTag("cosmicMuons") # for cosmicMuons
-
-process.out = cms.OutputModule("PoolOutputModule",
-  outputCommands = cms.untracked.vstring('drop *',
-        'keep *_dt4DSegments_*_*',
-        'keep *_cscSegments_*_*',
-        'keep *_rpcPointProducer_*_*',
-        'keep *_rpcRecHits_*_*',
-        'keep *_standAloneMuons_*_*',
-        'keep *_cosmicMuons_*_*',
-        'keep *_globalMuons_*_*'),
- fileName = cms.untracked.string('output.root')
+process.rpcPointProducer = cms.EDProducer('RPCPointProducer',
+  incldt = cms.bool(True),
+  inclcsc = cms.bool(True),
+  incltrack =  cms.bool(False),
+  debug = cms.bool(False),
```

Comment:
```suggestion
```
You can remove and just keep the default, which is `false`
Otherwise, make it Untracked as it should